docker/yb-voyager-docker

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
```

Comment:
We can simplify it a bit:
```
#!/bin/sh

export_dir=""
args=( "$@" )
i=0
while [ $i -lt $# ]
do
        w=${args[$i]}
        if [ "${w}" == '--export-dir' ]
        then
                j=$(( $i + 1))
                export_dir=${args[$j]}
                args[$j]='/export-dir'
        fi
        i=$(( $i + 1))
done
if [ -z ${export_dir} ]
then
        echo "error: required argument 'export-dir' not set"
        exit 1
fi
cmd="docker run -v ${export_dir}:/export-dir -it --rm --privileged --name voyager yb-voyager:latest yb-voyager ${args[*]}"
echo "$cmd"
```

Reply:
This will give error even while running `--help` for commands since we don't specify `export-dir` for them.